experimental/GModule/GaloisCohomology.jl

```diff
@@ -1081,15 +1081,15 @@ function Oscar.completion(I::IdelParent, P::NfAbsOrdIdl)
   mUp = I.L[p]
   mGp = I.D[p]
 
-  inj = Hecke.canonical_injection(I.M, p+1) #units are first
-  pro = Hecke.canonical_projection(I.M, p+1)
+  inj = canonical_injection(I.M, p+1) #units are first
+  pro = canonical_projection(I.M, p+1)
 
 
   @assert domain(inj) == codomain(pro)
 
   J = components(I.M)[p+1]
   if mKp.P == P #easy case
-    return Kp, mKp, mGp, mUp,  pro * Hecke.canonical_projection(J, 1) ,  Hecke.canonical_injection(J, 1)*inj
+    return Kp, mKp, mGp, mUp,  pro * canonical_projection(J, 1) ,  canonical_injection(J, 1)*inj
```

Comment:
Not your "fault", but while we are at it:
```suggestion
    return Kp, mKp, mGp, mUp, pro * canonical_projection(J, 1), canonical_injection(J, 1)*inj
```